jenkins/debian/debos/scripts/buster-cros-ec-tests.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+
+# Important: This script is run under QEMU
+
+set -e
+
+# Build-depends needed to build the test suites, they'll be removed later
+BUILD_DEPS="\
+    git \
+    python3-setuptools \
+    python3-pip \
+"
+
+apt-get install --no-install-recommends -y ${BUILD_DEPS}
+
+########################################################################
+# Build tests                                                          #
+########################################################################
+
+BUILDFILE=/test_suites.json
+echo '{  "tests_suites": [' >> $BUILDFILE
```

Comment:
Err, I meant this is only to start creating the file but then the git revision of cros-ec-tests should be added there as well, like in buster-igt.sh.